docs/getting-started/create-an-app.md

```diff
@@ -0,0 +1,81 @@
+# Backstage App
+
+If you want to get set up quickly with your own Backstage project you can create a Backstage App.
+
+A Backstage App is a monorepo setup with `lerna` that includes everything you need to run Backstage in your own environment.
+
+## Create an app
+
+If you have [`npx`](https://github.com/npm/npx) installed simply run:
+
+```bash
+npx @backstage/cli create-app
+```
+
+Or with `yarn`:
+
+```bash
+yarn add --dev @backstage/cli
+yarn run backstage-cli create-app
+```
+
+This will create a new Backstage App inside the current folder. The name of the app-folder is the name that was provided when prompted.
+
+<p align='center'>
+    <img src='https://github.com/spotify/backstage/raw/master/docs/getting-started/create-app_output.png' width='600' alt='create app'>
+</p>
+
+Inside that directory, it will generate all the files and folder structure needed for you to run your app.
+
+### Folder structure
+
+```
+app
+├── README.md
+├── lerna.json
+├── package.json
+├── prettier.config.js
+├── tsconfig.json
+├── packages
+│   └── app
+│       ├── package.json
+│       ├── tsconfig.json
+│       ├── public
+│       │   └──  ...
+│       └── src
+│           ├── App.test.tsx
+│           ├── App.tsx
+│           ├── index.tsx
+│           ├── plugins.ts
+│           └── setupTests.ts
+└── plugins
+    └── welcome
```

Comment:
Perhaps this should show "your-plugin-x" etc?

Reply:
This is what it generates right now

Reply:
I'm thinking that this list of files and folders is what `create-app` provides for you when you run it. Then when you use `create-plugin` we have in our docs that it adds it to the `plugins/` directory.